### PR TITLE
Police hats are now hats instead of helmets.

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Makes police hats a subtype of Warden's hats, so they still are security-level armor, without covering your hair like helmets do.
This was gone unnoticed because the item is very limited in-game. I've been in this place before https://github.com/tgstation/tgstation/pull/66872 ...

I also renamed the gravity generator updatepaths because it's been inconsistent with the rest of the updatepaths for a while now.

## Why It's Good For The Game

I hate it when a hats that don't cover your head end up turning you bald.

## Changelog

:cl:
fix: Police hats no longer turn you bald while worn, and no longer act like helmets, since they're caps.
/:cl:

### CODE REVIEW

**Comment by Kapu1178** [comment]
this works but you can also just remove the HIDEHAIR flag

**Comment by JohnFulpWillard** [comment]
> this works but you can also just remove the HIDEHAIR flag

I don't want to do that because I hate snowflake solutions.

File: tools/UpdatePaths/Scripts/69643_police_hat.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+/obj/item/clothing/head/helmet/police : /obj/item/clothing/head/warden/police
```
Comment (san7890) [inline tools/UpdatePaths/Scripts/69643_police_hat.txt:1]:
```suggestion
/obj/item/clothing/head/helmet/police : /obj/item/clothing/head/warden/police{@OLD}
```

Did you update the DMM manually?

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -191,7 +191,7 @@
 	resistance_flags = FIRE_PROOF | ACID_PROOF
 	dog_fashion = null
 
-/obj/item/clothing/head/helmet/police
+/obj/item/clothing/head/warden/police
```
Comment (ShizCalev) [inline code/modules/clothing/head/helmet.dm:194]: pssst.

helmet.dm

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Iamgoofball** [comment]
sorry bro im just gonna say it, it's more snowflake to repath it to warden helmet than to make it hide the hair

**Comment by ShizCalev** [comment]
image
image
image
balance change?

**Comment by JohnFulpWillard** [comment]
> sorry bro im just gonna say it, it's more snowflake to repath it to warden helmet than to make it hide the hair

The sprite is a hat, it's called a hat, I don't see why it should be treated like a helmet.
I also found out they share the same icon and icon state as the warden's default hat, so the only difference between the two was their name, and description.

**Pull request merged**